grails-app/views/report/showBinLocationReport.gsp

```diff
@@ -195,7 +195,9 @@
                 $('td:eq(1)', nRow).html('<a href="${request.contextPath}/inventoryItem/showStockCard/' + aData["id"] + '">' + aData["productCode"] + '</a>');
                 $('td:eq(2)', nRow).html('<a href="${request.contextPath}/inventoryItem/showStockCard/' + aData["id"] + '">' + aData["productName"] + ' ' +
                   renderHandlingIcons(aData) +  '</a>');
-                return nRow;
+                if (aData["lotStatus"] === "RECALLED") {
+                  $(nRow).removeClass().css('backgroundColor', '#ffcccb');
```

Comment:
Why is the `removeClass` here?

Reply:
@awalkowiak Because dataTable adds "sorting" class to all columns that are sorted and because of that background color is not applied to those columns.